.github/workflows/dart_ci.yml

```diff
@@ -34,19 +34,19 @@ jobs:
         if: always() && steps.install.outcome == 'success'
 
       - name: Validate formatting
-        run: pub run dart_dev format --check
+        run: dart format --set-exit-if-changed .
```

Comment:
This step is failing on stable because Dart 2.12.0 and Dart 2.14.2 have different opinions on how things should be formatted.

Reply:
I'm not sure if we should only check formatting on one of them?

Reply:
Yeah, that's a good question. I'm not sure if we have a norm through all of our repos, but I'd expect we'd want to run formatting on the one that we expect folks to be using and not worry about the latest version. I'm not sure though